utils.h

```diff
@@ -0,0 +1,55 @@
+/*  
+	  sensord - Sensor Interface for XCSoar Glide Computer - http://www.openvario.org/
+    Copyright (C) 2014  The openvario project
+    A detailed list of copyright holders can be found in the file "AUTHORS" 
+
+    This program is free software; you can redistribute it and/or 
+    modify it under the terms of the GNU General Public License 
+    as published by the Free Software Foundation; either version 3
+    of the License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, see <http://www.gnu.org/licenses/>.	
+*/
+
+/**
+ * access class for various utilities and types
+ */
+
+#ifndef VARIOD_UTILS
+#define VARIOD_UTILS
+
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "audiovario.h"
+#include "stf.h"
+#define NUM_FV  5 // maximum number of values per NMEA sentence
+
+class variod_utils{
```

Comment:
Does it really have to be a class? Judging from the name this is going to be just a set of assorted static functions with no state, so it can be simply a set of functions as well.

Reply:
Doesn't have to be a class. Maybe later?
Does it hurt to be a class?
A class doesn't cram everything into one name space hoping for the best.
I'd like to keep it that way unless there are good reasons not to.

Reply:
Well, the project was a pure C so far, AFAICT. This little thing turns it into a C++ project. To do it properly, you would need to rename this file to .hpp or hxx. And also rename files that include this header to .cpp or .cxx, because technically they are no longer C programs anymore. 

Is it worth it?

Reply:
I'll make it plain C in the next round after this PR got merged.